dub.json

```diff
@@ -110,19 +111,6 @@
             "targetName": "agora-client",
             "mainSourceFile": "source/agora/cli/client/main.d",
             "excludedSourceFiles": [
-                "source/agora/api/Validator.d",
-                "source/agora/common/Config.d",
-                "source/agora/common/SCPHash.d",
-                "source/agora/consensus/Quorum.d",
-                "source/agora/network/Client.d",
-                "source/agora/network/Manager.d",
-                "source/agora/node/*",
-                "source/agora/flash/*",
-                "source/agora/utils/SCPPrettyPrinter.d",
-                "source/scpp/*",
-                "source/scpd/*",
-                "source/agora/consensus/protocol/Nominator.d",
-                "source/agora/consensus/SCPEnvelopeStore.d",
```

Comment:
Because it caused compile errors and when I tried to selectively remove some files from the list (and not all), then a different compile time error message appeared. After trying for 10 minutes I gave it up... All of this becomes very fiddly and brittle just to try to reduce compile time.

Reply:
It wasn't to remove compile time, but to remove dependency. But I agree it's brittle. There has been quite a few improvements in both DUB and the code base since this was first written, I'll revisit to see if there's an obvious path forward. If not, I'm not too bothered by the diff, but wanted to understand where it came from.